src/applications/edu-benefits/feedback-tool/config/form.js

```diff
@@ -383,6 +383,9 @@ const formConfig = {
                     street2: {
                       'ui:title': 'Address line 2'
                     },
+                    street3: {
```

Comment:
We may need to transform this on the FE until the backend has included this transformation in its transformation list.

Reply:
Nevermind, it looks like this work is already done. Way to go @jholton.
https://github.com/department-of-veterans-affairs/vets-api/blob/f367cbde99b9fc812d6bd29bab06b0b312a69ea2/app/models/gi_bill_feedback.rb

Reply:
@cehsu I only see that transform happening [here](https://github.com/department-of-veterans-affairs/vets-api/blob/f367cbde99b9fc812d6bd29bab06b0b312a69ea2/app/models/gi_bill_feedback.rb#L34), which is no longer gonna run.

Reply:
Since the backend needs to be updated before we can release this I don't think we should bandaid it on the frontend.

Reply:
Should we create a backend ticket to update the international school address transformation so that street3 is included when condensing international addresses onto only the street1 and street2 fields?

Reply:
ticket: https://app.zenhub.com/workspace/o/department-of-veterans-affairs/vets.gov-team/issues/13074